packages/ember-simple-auth/package.json

```diff
@@ -49,7 +50,7 @@
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "4.0.2",
     "ember-cli-yuidoc": "^0.9.1",
-    "ember-data": "~3.18.0",
+    "ember-data": "~4.4.0",
```

Comment:
Hmm, now that I think about it, ember-data likely can be removed from ESA.
Will do separately.